pkg/securityscan/controller.go

```diff
@@ -38,11 +39,13 @@ type Controller struct {
 	xcs              *kubeapiext.Clientset
 	coreFactory      *corectl.Factory
 	batchFactory     *batchctl.Factory
+	appsFactory      *appsctl.Factory
 	cisFactory       *cisoperatorctl.Factory
 	apply            apply.Apply
 	monitoringClient v1monitoringclient.MonitoringV1Interface
 
-	mu *sync.Mutex
+	mu              *sync.Mutex
+	CurrentScanName string
```

Comment:
It doesn't look like this is used outside the controller, does it need to be exported

Reply:
done